File: test/fixtures/stdlib/string/transform/replace.expected.py
(lambda _: kAssert("hello world".replace("world", "there", 1) == "hello there"))(None)
(lambda _: kAssert("abab".replace("ab", "x", 1) == "xab"))(None)
(lambda _: kAssert("abab".replace("ab", "x") == "xx"))(None)

File: test/fixtures/stdlib/string/transform/replace.expected.js
(function(_) { return (function() { if (!("hello world".replace("world", "there") == "hello there")) throw new Error("Assertion failed"); return true; })(); })(null);
(function(_) { return (function() { if (!("abab".replace("ab", "x") == "xab")) throw new Error("Assertion failed"); return true; })(); })(null);
(function(_) { return (function() { if (!("abab".replaceAll("ab", "x") == "xx")) throw new Error("Assertion failed"); return true; })(); })(null);
